### PR TITLE
Add Eva crate

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

**Media**

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added crate with two prestigious EVA hardsuits to cargo catalog

### CODE REVIEW

File: Resources/Locale/en-US/prototypes/catalog/fills/crates/engineering-crates.ftl [hunk 1]
```diff
@@ -30,3 +30,6 @@ ent-CrateEngineeringMiniJetpack = Mini jetpack crate
 
 ent-CrateAirlockKit = Airlock kit
     .desc = A kit for building 6 airlocks, doesn't include tools.
+
+ent-CrateEvaKit = Eva kit
```
Comment (ficcialfaint) [inline Resources/Locale/en-US/prototypes/catalog/fills/crates/engineering-crates.ftl:34]:
```suggestion
ent-CrateEvaKit = EVA kit
```

File: Resources/Locale/en-US/prototypes/catalog/fills/crates/engineering-crates.ftl [hunk 2]
```diff
@@ -30,3 +30,6 @@ ent-CrateEngineeringMiniJetpack = Mini jetpack crate
...
+    .desc = A set consisting of two prestigious EVA suits and helmets, tanks not included.
```
Comment (ficcialfaint) [inline Resources/Locale/en-US/prototypes/catalog/fills/crates/engineering-crates.ftl:35]:
```suggestion
    .desc = A set consisting of two prestigious EVA suits and helmets.
```

**Comment by ficcialfaint** [comment]
In CL: `Eva` --> `EVA`

File: Resources/Locale/en-US/prototypes/catalog/fills/crates/engineering-crates.ftl (hunk 2 above)
Comment (lzk228) [reply Resources/Locale/en-US/prototypes/catalog/fills/crates/engineering-crates.ftl:35]: for what?

Comment (ficcialfaint) [reply Resources/Locale/en-US/prototypes/catalog/fills/crates/engineering-crates.ftl:35]: why would it be mentioned here?

Comment (ficcialfaint) [reply Resources/Locale/en-US/prototypes/catalog/fills/crates/engineering-crates.ftl:35]: but the description literally says it contains only helmets and suits, is it for autistic players who cannot get it from reading for the first time?

Comment (deltanedas) [reply Resources/Locale/en-US/prototypes/catalog/fills/crates/engineering-crates.ftl:35]: if you are gonna say theres no tanks include that there are no magboots or breath masks either, they are in the eva room too.

**Pull request merged**